docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -486,18 +484,15 @@ Development builds typically display an onboarding welcome screen when an app is
 An example of such a Detox test is shown below. Full example code is available on the [eas-tests-example](https://github.com/expo/eas-tests-example) repository.
 
 <Collapsible summary="e2e/homeScreen.e2e.js">
-
 ```js
 /* @info New line */
-const { openAppForDebugBuild } = require('./utils/openAppForDebugBuild');
+const { openApp } = require('./utils/openApp');
 /* @end */
 
 describe('Home screen', () => {
   beforeEach(async () => {
-    await device.launchApp({
-      newInstance: true,
-    });
-    /* @info New line */ await openAppForDebugBuild(); /* @end */
+    // await device.launchApp();
```

Comment:
This should be a copy-paste from https://github.com/expo/eas-tests-example I think this is really important to not confuse users.
```suggestion
```